doc/classes/RichTextLabel.xml

```diff
@@ -111,7 +111,7 @@
 			<argument index="0" name="align" type="int" enum="RichTextLabel.Align">
 			</argument>
 			<description>
-				Adds a [code][right][/code] tag to the tag stack.
+				Adds a [code]Align[/code] tag to the tag stack. See enum [code]Align[/code] constants.
```

Comment:
All other `push_` methods are document with the actual tag name that they add, so `Add a Align tag` seems weird. You can also use `[enum Align]` to reference enums directly. So someting like `Adds an alignment tag based on the given [code]align[/code] value. See [enum Align] for possible values.`.

Reply:
Bump.